src/dnx.coreclr.unix/dnx.coreclr.cpp

```diff
@@ -257,11 +257,23 @@ std::string get_os_version()
             {
                 if (line.compare(0, qualifier.length(), qualifier) == 0)
                 {
+                    auto value = line.substr(qualifier.length());
+                    if (value.length() == 0)
```

Comment:
You're still not handling `value.length() == 1`. The check should be `<2` instead of `==0`